docs/scriptworkers-FAQ.md

```diff
@@ -34,15 +34,20 @@ on your local machine:
 find the latest image pushed
 * pull that image locally by `docker pull mozilla/releng-$script:production--$timestamp-$hash`
 * use `pass` to define some dummy values replacing the one it'd expect in production (e.g. a file called `local-prod`)
```

Comment:
Adjusting a bit the docs since I found it incomplete - https://scriptworker-scripts.readthedocs.io/en/latest/scriptworkers-FAQ.html